master/buildbot/status/web/hooks/github.py

```diff
@@ -216,7 +217,10 @@ def getChanges(request, options=None):
         request
             the http request object
     """
-    if options is None:
+    if options is None or (isinstance(options, bool) and options):
+        if options is not None:
+            warnings.warn('Use of None for GitHub hook is deprecated.  '
```

Comment:
I don't get how that line correlates with
"warnings.warn('Use of None for GitHub hook is deprecated.  '
                          'Please see the documentation for possible values')"
if options is None:
    warnings.warn('Use of None for GitHub hook is deprecated.  '
                            'Please see the documentation for possible values')

seems more logical, isn't it ?